### PR TITLE
CAMEL-11274- add extensibility to ReloadStrategySupport in case needed

### PR DESCRIPTION
**Pull request by onders86**

### CODE REVIEW

File: camel-core/src/main/java/org/apache/camel/support/ReloadStrategySupport.java [hunk 1]
```diff
@@ -181,6 +181,16 @@ public int getReloadCounter() {
     public int getFailedCounter() {
         return failed;
     }
+    
+    @ManagedOperation(description = "Set succeeded count on xml reload in case overriden")
```
Comment (davsclaus) [inline camel-core/src/main/java/org/apache/camel/support/ReloadStrategySupport.java]: I dont think these should be JMX write operations, eg remove the annotation. There can still be a setter for extensions

Comment (onders86) [inline camel-core/src/main/java/org/apache/camel/support/ReloadStrategySupport.java]: this PR is not really needed IMHO as described in the issue. Do you mind if i close it and mark the issue "Won't fix" or do you think we can keep like this and resolve it?

**Comment by davsclaus** [comment]
Thanks for the PR it has been merged. Do you mind closing this?

**Comment by onders86** [comment]
thanks for accepting. closing.

**Pull request closed without merging**